doc/fill_gpi.py

```diff
@@ -66,7 +66,7 @@ def signature(func, name=None):
     Returns: a string representing its signature as would be written in code
 
     """
-    args, varargs, varkw, defaults = inspect.getargspec(func)
+    args, varargs, varkw, defaults, kwonlyargs, kwonlydefaults, annotations = inspect.getfullargspec(func)
```

Comment:
If we are not using these extra variables it may be better to replace them with `_`

Reply:
Hey, will

```python
args, varargs, varkw, defaults, *_ = inspect.getfullargspec(func)
```

be better?

Reply:
Yes

Reply:
Hey updated it. Also tested the update on python 3.7 and 3.12.